SquadForger/View/SquadView.xaml

```diff
@@ -25,8 +25,8 @@
                 <Button Content="Select file" Margin="10" Width="100" Command="{Binding SelectFileCommand}"></Button>
             </TextBlock>
             <TextBlock VerticalAlignment="Center">
-                <TextBox TextWrapping="Wrap" VerticalScrollBarVisibility="Auto" Text="Add teams manually separated by ," Height="200" Width="200" Margin="10"></TextBox>
-                <Button Content="Add" Margin="10" Width="50" Height="30"></Button>
+                <TextBox TextWrapping="Wrap" VerticalScrollBarVisibility="Auto" Text="{Binding TeamsInput}" Height="200" Width="200" Margin="10"></TextBox>
+                <Button Content="Add" Margin="10" Width="50" Height="30" Command="{Binding AddTeamsCommand}"></Button>
                 <Button Content="Clear" Margin="10" Width="50" Height="30"></Button>
```

Comment:
I feel like clear button should do something with this PR since currently there is no way to remove teams from the list.